webapi/src/main/scala/org/knora/webapi/messages/util/ValueUtilV1.scala

```diff
@@ -343,6 +343,8 @@ final case class ValueUtilV1Live(
       "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"         -> "XLSX",
       "application/xml"                                                           -> "XML",
       "text/xml"                                                                  -> "XML",
+      "text/xml"                                                                  -> "ODD",
```

Comment:
I think the recommended MIME type would be: application/tei+xml.

See the format description in the pronom database:
https://www.nationalarchives.gov.uk/PRONOM/Format/proFormatSearch.aspx?status=detailReport&id=2297
https://www.nationalarchives.gov.uk/PRONOM/Format/proFormatSearch.aspx?status=detailReport&id=2298
https://www.nationalarchives.gov.uk/PRONOM/Format/proFormatSearch.aspx?status=detailReport&id=2299
https://www.nationalarchives.gov.uk/PRONOM/Format/proFormatSearch.aspx?status=detailReport&id=2300

Reply:
Thanks for the hint. It confuses me now, because this PR has been started over 6 months ago and I slightly recall there was something about it that I put for some reason `text/xml`, but I son't really know was was the reason behind it. This simple type is also [used in SIPI](https://github.com/dasch-swiss/sipi/blob/08ad78e06699c3e94cd9388937fcd7b895e22666/shttps/Parsing.cpp#L81):

Reply:
Indeed is not. I'm glad I have got this working for months of waiting!